pkg/grpc/api_test.go

```diff
@@ -2822,15 +2930,44 @@ func TestGetConfigurationAlpha1(t *testing.T) {
 			},
 		})
 
+		assert.NoError(t, err)
+		assert.NotNil(t, r.Items)
+		assert.Len(t, r.Items, 1)
+		assert.Equal(t, "val1", r.Items["key1"].Value)
+	})
+	t.Run("get configuration item", func(t *testing.T) {
+		compStore := compstore.New()
+		compStore.AddConfiguration("store1", &mockConfigStore{})
+		server, lis := startDaprAPIServer(&api{
+			id: "fakeAPI",
+			UniversalAPI: &universalapi.UniversalAPI{
+				AppID:     "fakeAPI",
+				CompStore: compStore,
+			},
+			resiliency: resiliency.New(nil),
+		}, "")
+		defer server.Stop()
+
+		clientConn := createTestClient(lis)
+		defer clientConn.Close()
+
+		client := runtimev1pb.NewDaprClient(clientConn)
+		r, err := client.GetConfiguration(context.Background(), &runtimev1pb.GetConfigurationRequest{
+			StoreName: "store1",
+			Keys: []string{
+				"key1",
+			},
+		})
+
 		assert.NoError(t, err)
 		assert.NotNil(t, r.Items)
 		assert.Len(t, r.Items, 1)
 		assert.Equal(t, "val1", r.Items["key1"].Value)
 	})
 }
 
-func TestSubscribeConfigurationAlpha1(t *testing.T) {
-	t.Run("get configuration item", func(t *testing.T) {
+func TestSubscribeConfiguration2(t *testing.T) {
```

Comment:
TestSubscribeConfigurationAPI may be better.

Reply:
Done